src/components/ui/Footer.js

```diff
@@ -37,15 +41,15 @@ class Footer extends React.Component {
         return(
           // TODO (Add icon)
             <Wrapper>
-                <Row>
+                <Row onClick={() => window.location.href=githubURL}>
                     {'Contribute on'}
 
-                    <Icon src={'../static/img/footer/Github.svg'} />
+                    <Icon style={{paddingBottom:"2px"}} src={'../static/img/footer/Github.svg'} />
```

Comment:
is there any reason that style has double bracket?

Reply:
first bracket is to use javascript object in jsx grammar and second means an object. In detail, 
const style = {paddingBottom:"2px};
<Icon style= {style} >